helm/prometheus-rules/templates/alerting-rules/certificate.all.rules.yml

```diff
@@ -17,10 +17,10 @@ spec:
       expr: (cert_exporter_secret_not_after{name=~"kiam.*"} - time()) < 2 * 7 * 24 * 60 * 60
       for: 5m
       labels:
-        area: managedapps
+        area: kaas
         cancel_if_outside_working_hours: "true"
         severity: page
-        team: cabbage
+        team: phoenix
```

Comment:
Should these all not be `phoenix` ? 

Hydra doesn't currently have alerting finished. We're still on the `phoenix` on-call.

Reply:
that works for me

Reply:
I was asking because I wasn't sure. This PR is how I found out we (Hydra) apparently own cert-manager 😆  Which seems an odd choice to me.